### DISCUSSION TITLE
The include tag seems work only when preceded by whitespace

### DISCUSSION THREAD

**Discussion by ialbert** [quarto-dev/quarto-cli#1766] [Q&A] [answered]
The `include` tag seems to work only when preceded by whitespace. Example code:
```
Including on a new line works:

{{< include _date.md >}}

this also works:

     {{< include _date.md >}}

This does not work: {{< include _date.md >}}
```

the rendered HTML output is:

```
Including on a new line works:

May 26, 2022

this also works:

May 26, 2022

This does not work:
```

is that by design? Thanks.

**Comment by cscheid** [comment, marked as answer]
Thanks for the report! I'm using https://github.com/quarto-dev/quarto-cli/issues/1767 to track. The short answer is that it's an accident of previous implementations of shortcodes. We'd like to change that, but it will take a while, so we don't have a firm timeline on this yet.

**Reply by sckott** [reply]
Is there any hack I can use in the meantime to make this work?
